main/lsp/test/lsp_preprocessor_test.cc

```diff
@@ -23,11 +26,11 @@ options::Options makeOptions(string_view rootPath) {
     return opts;
 }
 
-static auto nullSink = make_shared<spd::sinks::null_sink_mt>();
-static auto logger = make_shared<spd::logger>("console", nullSink);
-static auto typeErrorsConsole = make_shared<spd::logger>("typeDiagnostics", nullSink);
-static auto nullOpts = makeOptions("");
-static auto workers = WorkerPool::create(0, *logger);
```

Comment:
Clang-tidy informed me that the `static` here is redundant in an anonymous namespace.